README.md

```diff
@@ -84,30 +72,55 @@ download methods were (1) `dx cat`, and (2) `cat` from a dxfuse mount point.
+
+Creating and writing to files is allowed when dxfuse is mounted with the `-limitedWrite` flag.
+Writing to files is **append only**. Any non-sequential writes will return `ENOTSUP`. Seeking or reading operations are not permitted while a file is being written.
 
 ## Supported operations
 
-`-limitedWrite` mode also enables the following operations: rename (mv), unlink (rm), mkdir (see below), and rmdir (empty folders only). Rewriting existing files is not permitted, nor is truncating existing files. 
+`-limitedWrite` mode enables the following operations: rename (mv, see [below](#rename-behavior)), unlink (rm), mkdir (see [below](#mkdir-behavior)), and rmdir (empty folders only). Rewriting existing files is not permitted, nor is truncating existing files.
 
 ### mkdir behavior
 
-All `mkdir` operations via dxfuse are treated as `mkdir -p`. This is because dxfuse does not present the realtime state of the project. Folders can be created outside of dxfuse (or in another dxfuse process), and therefore not be visible to the current running dxfuse. A subsequent `mkdir` --> `project-xxxx/newFolder` would return a 422 error, because dxfuse did not know the directory already exists. This design is due to spark behavior where multiple worker nodes sometimes attempt to create the same output directory. 
+All `mkdir` operations via dxfuse are treated as `mkdir -p`. This is because dxfuse does not present the realtime state of the project. Folders can be created outside of dxfuse (or in another dxfuse process), and therefore not be visible to the current running dxfuse. A subsequent `mkdir` --> `project-xxxx/newFolder` would return a 422 error, because dxfuse did not know the directory already exists. This design is due to spark behavior where multiple worker nodes sometimes attempt to create the same output directory.
+
+### rename behavior
+
+Rename does not allow removing the target file or directory because DNAnexus API does not support this functionality.
+XXX moved here from limitations.  Still relevant and correct?
 
 ## File upload and closing
 
-Each file open for writing is allocated a 96MiB write buffer in memory, which is uploaded as a file part when full. Currently dxfuse will upload up to 4 parts in parallel. 
-The last part upload and `file-xxxx/close` DNAx operation is called only when a file descriptor is closed and the `FlushFile` fuse operation is triggered.
+Each dxfuse file open for writing is allocated a 96MiB write buffer in memory, which is uploaded as a DNAnexus file part when full. dxfuse will upload up to 4 parts in parallel.
+
+XXX 4 parts per file or 4 parts across all files backed by dxfuse?
+
+The upload of the last DNAnexus file part and the call of `file-xxxx/close` DNAnexus API operation are performed by dxfuse only when the OS process that created the OS file descriptor closes the OS file descriptor, triggering `FlushFile` fuse operation.
 
```

Comment:
We should add a section regarding FlusFile operations ignored from forked processes as detailed in DEVEX-1948. File descriptors must be closed by the original process which opened them.

Reply:
I'll leave the current minimal mention in place for now.  We can add a more detailed explanation later